### PR TITLE
feat: revived user provided UBO block

### PR DESCRIPTION
**Pull request by tomer-shvadron**
### Description
* Revived User-Provided UBO block
* Fixed sub-title case

image

### CODE REVIEW

File: apps/backoffice-v2/src/lib/blocks/hooks/useAssociatedCompaniesInformationBlock/useAssociatedCompaniesInformationBlock.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const useAssociatedCompaniesInformationBlock = (workflows: Array<TWorkflo
                 })
                 .addCell({
                   type: 'subheading',
-                  value: 'User-provided data',
+                  value: 'User-Provided Data',
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/lib/blocks/hooks/useAssociatedCompaniesInformationBlock/useAssociatedCompaniesInformationBlock.tsx:33]: Are you sure that's how hyphenated words are capitalized?

File: apps/backoffice-v2/src/lib/blocks/hooks/useUbosUserProvidedBlock/useUbosUserProvidedBlock.tsx
Comment (Omri-Levy) [inline apps/backoffice-v2/src/lib/blocks/hooks/useUbosUserProvidedBlock/useUbosUserProvidedBlock.tsx:1]: Missing percentage of ownership column.

File: apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic.tsx [hunk 2]
```diff
@@ -232,7 +234,7 @@ export const useDefaultBlocksLogic = () => {
   });
 
   const entityInfoBlock = useEntityInfoBlock({
-    entity: workflow?.context?.entity,
+    entity: workflow?.context?.entity ?? {},
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic.tsx]: An empty object would make the data checks harder as opposed to undefined.

File: apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic.tsx [hunk 3]
```diff
@@ -361,7 +368,8 @@ export const useDefaultBlocksLogic = () => {
       ...directorsUserProvidedBlock,
       ...directorsRegistryProvidedBlock,
       ...directorsDocumentsBlocks,
-      ...ubosBlock,
+      ...ubosUserProvidedBlock,
+      ...ubosRegistryProvidedBlock,
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic.tsx]: Sync with @nitzanballerine over the blocks order.

File: apps/backoffice-v2/src/lib/blocks/hooks/useAssociatedCompaniesInformationBlock/useAssociatedCompaniesInformationBlock.tsx (hunk 1 above)
Comment (tomer-shvadron) [reply apps/backoffice-v2/src/lib/blocks/hooks/useAssociatedCompaniesInformationBlock/useAssociatedCompaniesInformationBlock.tsx:33]: I'm sure that's how Nitzan wanted it 🤷

File: apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic.tsx (hunk 2 above)
Comment (tomer-shvadron) [reply apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic.tsx]: Fair, so it's just a TS error right now (as `useEntityInfoBlock` expects `entity` to not be undefined)

File: apps/backoffice-v2/src/lib/blocks/hooks/useUbosUserProvidedBlock/useUbosUserProvidedBlock.tsx
Comment (tomer-shvadron) [reply apps/backoffice-v2/src/lib/blocks/hooks/useUbosUserProvidedBlock/useUbosUserProvidedBlock.tsx:1]: Added

**Review by Omri-Levy — APPROVED** [review]

**Pull request merged**